stor/tests/cassettes_py3/TestLoginAuth/test_login_auth.yaml

```diff
@@ -239,18 +239,18 @@ interactions:
           ZHhweS8wLjI2NS4wLTgwLWc0YjYwMTY4NyAoRGFyd2luLTE4LjAuMC14ODZfNjQtaTM4Ni02NGJp
           dCk=
     method: POST
-    uri: https://api.dnanexus.com/project-FPGB9600yXpPxYfQJq0Qvb5G/destroy
+    uri: https://api.dnanexus.com/project-FPJ07G00JKbKjyyG4xxQJvKQ/destroy
   response:
-    body: {string: '{"id":"project-FPGB9600yXpPxYfQJq0Qvb5G"}'}
+    body: {string: '{"id":"project-FPJ07G00JKbKjyyG4xxQJvKQ"}'}
     headers:
       Connection: [keep-alive]
       Content-Length: ['41']
       Content-Type: [application/json; charset=utf-8]
-      Date: ['Tue, 30 Oct 2018 18:36:12 GMT']
+      Date: ['Wed, 31 Oct 2018 19:34:10 GMT']
       Server: [nginx]
       X-Content-Type-Options: [nosniff]
       X-Powered-By: [Express]
-      X-Request-ID: [1540924569244-887656]
+      X-Request-ID: [1541014448711-910820]
       X-Upgrade-Info: ['A recommended update (v0.267.0) is available for your client/SDK.
```

Comment:
random comment: does this need a freezegun and is that why it didn't work?

Reply:
Nah, pretty sure that is okay. vcr matches only on uri and method, not the time of request or anything (hence, we don't need freezegun in general). I actually have no idea why it started failing in local. The only thing concerning the test is that we were talking about it :| . It seems to still work with Travis though, so I'm leaving it be for now.